README.md

```diff
@@ -62,27 +64,26 @@ If you use Terraform to manage your infrastructure, MongoDB for VS Code helps yo
+| `mdb.useDefaultTemplateForPlayground`                              	| Choose whether to use the default template for playground files or to start with an empty playground editor.                                                                                                                         	| `true`                                                                                                                     	|
+| `mdb.uniqueObjectIdPerCursor`                                      	| The default behavior is to generate a single ObjectId and insert it on all cursors. Set to true to generate a unique ObjectId per cursor instead.                                                                                    	| `false`                                                                                                                    	|
+| `mdb.sendTelemetry`                                                	| Opt-in and opt-out for diagnostic and telemetry collection.                                                                                                                                                                          	| `true`                                                                                                                     	|
 ## Additional Settings
 
 ### Autocomplete
 
-_These global settings affect how MongoDB for VS Code provides intelligent autocomplete inside snippets and string literals (off by default). Changing the default configuration may affect the behavior and performance of other extensions and of VS Code itself. If you do not change the default settings, you can still trigger intelligent autocomplete inside a snippet or string literal with `Ctrl+Space`._
+_Note: these global settings affect how MongoDB for VS Code provides intelligent autocomplete inside snippets and string literals (off by default). Changing the default configuration may affect the behavior and performance of other extensions and of VS Code itself. If you do not change the default settings, you can still trigger intelligent autocomplete inside a snippet or string literal with `Ctrl+Space`._
 
 - `editor.suggest.snippetsPreventQuickSuggestions`: By default, VS Code prevents code completion in snippet mode (editing placeholders in inserted code). Setting this to `false` allows snippet (eg. `$match`, `$addFields`) and field completion based on the document schema for the `db.collection.aggregate()` expressions.
 - `editor.quickSuggestions`: By default, VS Code prevents code completion inside string literals. To enable database names completions for `use('dbName')` expression use this configuration:
```

Comment:
@alenakhineika is this still true or just old behavior? I noticed that I have it `off` for strings and still I get autocomplete.

Reply:
Well, the setting still exists in vscode but it also does not work for me. I would keep it for a while and see if they remove or fix it :)